frontend/public/components/utils/vertnav.jsx

```diff
@@ -61,6 +61,11 @@ export const navFactory = {
     href: 'environment',
     name: 'Environment',
     component: component,
+  }),
+  clusterServiceClasses: component => ({
+    href: 'clusterServiceClasses',
+    name: 'Cluster Service Classes',
```

Comment:
We probably want to change this to `serviceClasses` and `name: 'Service Classes'`. Eventually we need to handle namespaced service classes as well. This would be consistent with our labels for role bindings (which have cluster scoped resources and namespaced resources).